docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+# Breaking Changes for PowerShell 6.0
+
+## Engine/language changes
+
+### Rename `powershell.exe` to `pwsh.exe` [#5101](https://github.com/PowerShell/PowerShell/issues/5101)
+
+In order to give users a deterministic way to call PowerShell Core on Windows (as opposed to Windows PowerShell),
+the PowerShell Core binary was changed to `pwsh.exe` on Windows and `pwsh` on non-Windows platforms.
+
+Additionally, this was done to reduce the typing time to open PowerShell,
+especially for non-Windows users who are used to three or four letter shell names.
+
+In the future, some kind of alias or symbolic link for `powershell` may be added to non-Windows platforms.
+
+### Don't insert line breaks to output (except for tables) [#5193](https://github.com/PowerShell/PowerShell/issues/5193)
+Previously, output was aligned to the width of the console and linebreak were added at the end width of the console, meaning the output doesn't get reformatted as expected if the terminal is resized.
+This change was not applied to tables, as the linebreaks are necessary to keep the columns aligned.
+
+### Skip null-element check for collections with a value-type element type [#5432](https://github.com/PowerShell/PowerShell/issues/5432)
+For the `Mandatory` parameter and `ValidateNotNull` and `ValidateNotNullOrEmpty` attributes, skip the null-element check if the collection's element type is value type.  
+
+### Change `$OutputEncoding` to use `iso-8859-1` encoding rather than ASCII [#5369](https://github.com/PowerShell/PowerShell/issues/5369)
+The previous encoding, ASCII (7-bit), would result in incorrect alteration of the output in some cases.  This change is to make UTF-8 NoBOM default, which preserves unicode output with an encoding supported by most tools and operating systems.
+
+### Remove `AllScope` from most default aliases [#5268](https://github.com/PowerShell/PowerShell/issues/5268)
+To speed up scope creation, `AllScope` was removed from most default aliases.
+`AllScope` was left for a few frequently used aliases where the lookup was faster.
+
+### Don't wrap `stderr` as `ErrorRecord` [#5190](https://github.com/PowerShell/PowerShell/issues/5190)
+
+Keep stderr as string and don't wrap as an `ErrorRecord`.
+This means that errors from running `pwsh.exe` within PowerShell Core are now just `stderr` strings even though they visually look like ErrorRecords
+This change my require you to change how your tests handle native commmand errors.
+
+### `-Verbose` and `-Debug` no longer overrides `$ErrorActionPreference` [#5113](https://github.com/PowerShell/PowerShell/issues/5113)
+
+Previously, if `-Verbose` or `-Debug` were specified, it overrode the behavior of `$ErrorActionPreference`.
```

Comment:
I think this could go either way, since there are 2 parameters specified I used there plural `were`.  But `was` would be appropriate because the parameters can be used individually.  In this case, I think `were` is correct since we are listing multiple parameters.

Reply:
Ok